tests/build_/test_arrangements.py

```diff
@@ -1703,3 +1729,319 @@ def test_export_operator_manifests(self, osbs, extract_platform):
+
+        expected_args = {
+            'load_exported_image': True,
+            'publish': False,
+        }
+
+        assert args == expected_args
+
+    @pytest.mark.parametrize('osbs', [OSBS_WITH_PULP_PARAMS], indirect=True)  # noqa:F811
+    def test_pulp_tag(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        get_plugin(plugins, 'postbuild_plugins', PLUGIN_PULP_TAG_KEY)
+        with pytest.raises(KeyError):
+            plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_PULP_TAG_KEY, 'args')
+
+    @pytest.mark.parametrize('osbs', [OSBS_WITH_PULP_PARAMS], indirect=True)  # noqa:F811
+    def test_pulp_sync(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_PULP_SYNC_KEY, 'args')
+        expected_args = {
+            'publish': False,
+        }
+
+        assert args == expected_args
+
+    @pytest.mark.parametrize('osbs, use_pulp', [  # noqa:F811
+        ({'platform_descriptors': None, 'additional_config': None, 'kwargs': None}, False),
+        (OSBS_WITH_PULP_PARAMS, True)], indirect=['osbs'])
+    def test_pulp_publish(self, osbs, use_pulp):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        with pytest.raises(NoSuchPluginException):
+            get_plugin(plugins, 'postbuild_plugins', PLUGIN_PULP_PUBLISH_KEY)
+
+    def test_pulp_pull(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'exit_plugins', PLUGIN_PULP_PULL_KEY, 'args')
+        expected_args = {'insecure': True}
+        assert args == expected_args
+
+    @pytest.mark.parametrize('osbs, with_group', [  # noqa:F811
+        ({'platform_descriptors': {'x86_64': {'architecture': 'amd64'}},
+            'additional_config': get_pulp_additional_config(),
+            'kwargs': {'registry_uri': 'registry.example.com/v2'}}, False),
+        ({'platform_descriptors': {'x86_64': {'architecture': 'amd64'}},
+            'additional_config': get_pulp_additional_config(with_group=True),
+            'kwargs': {'registry_uri': 'registry.example.com/v2'}}, True)],
+        indirect=['osbs'])
+    def test_group_manifests(self, osbs, with_group):
```

Comment:
`with_group` seems to be unused here, same concern as `use_pulp` above.

Reply:
Thanks, fixed.